haxe.html.markdown

```diff
@@ -383,11 +395,7 @@ class LearnHaxe3{
         */
 
         // if statements
-        var k = if (true){
-            10;
-        } else {
-            20;
-        }
+        var k = if (true) 10 else 20;
```

Comment:
It's somehow not obvious that one can use more lightweight syntax without curly braces. I think this guide also need to mention that even functions don't require `{}`, but maybe later.